### PR TITLE
Security: How to Build a Login Form: sync with recent Maker Bundle changes

### PR DESCRIPTION
**Pull request by mhujer**
Related to #11265

### CODE REVIEW

File: security/form_login_setup.rst [hunk 1]
```diff
@@ -279,6 +302,8 @@ a traditional HTML form that submits to ``/login``:
                     guard:
                         authenticators:
                             - App\Security\LoginFormAuthenticator
+                    logout:
+                        path: app_logout
 
     .. code-block:: xml
```
Comment (mhujer) [inline security/form_login_setup.rst:328]: XML and PHP config blocks should be updated as well, but I'm not sure about the correct syntax as I'm not using them

**Review by abdounikarim — APPROVED** [review]

File: security/form_login_setup.rst [hunk 2]
```diff
@@ -137,11 +146,17 @@ a traditional HTML form that submits to ``/login``:
             <div class="alert alert-danger">{{ error.messageKey|trans(error.messageData, 'security') }}</div>
         {% endif %}
 
+        {% if app.user %}
```
Comment (HeahDude) [inline security/form_login_setup.rst]: This is not needed since the controller rendering this route redirects if there is an authenticated user.

Comment (mhujer) [reply security/form_login_setup.rst]: It comes from https://github.com/symfony/maker-bundle/blob/f7dd84691dbf399137a5caafc90f419801d09e5f/src/Resources/skeleton/authenticator/login_form.tpl.php#L12

The goal of the PR was to sync the docs with the code generated by the Maker Bundle, so if it is not necessary, it should probably be changed there first?

Comment (HeahDude) [reply security/form_login_setup.rst]: Indeed :)

**Comment by wouterj** [comment]
Thank you @mhujer. I'm sorry for the long wait.. The good news: This PR is merged now!

I've updated the XML and PHP examples after the merge in 8a0c09efe

**Comment by mhujer** [comment]
@wouterj Thank you!

**Pull request merged**